mercadopago/px-ios
# (Changed) -  Renamed PaymentService and connected PX_ENVIRONMENT

## What have changed?

- Renamed PaymentService to CheckoutService

- Connected PX_ENVIRONMENT with RequestInfos for environment selection

## How to test:

Set PX_ENVIRONMENT on Info.plist->Information property list, choose between [alpha, beta, gamma, prod] and check for proper API calls
